lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php

```diff
@@ -36,34 +36,28 @@ public function convertException($message, DriverException $exception)
                 return new Exception\LockWaitTimeoutException($message, $exception);
             case '1050':
                 return new Exception\TableExistsException($message, $exception);
-
```

Comment:
I'd like these new lines to be kept. They visually separate `case`s from each other.

Reply:
If I kept them I will be breaking 

SlevomatCodingStandard.ControlStructures.ControlStructureSpacing.IncorrectLinesCountAfterLastControlStructure

Reply:
That's a bummer. I'd say it's an oversight on the standard side. Would you mind filing an issue? If `return` is part of a `case` followed by another `case`, the new line should be allowed.

For the time being, please revert these changes and disable the rule.